cmd/create/cluster/cmd.go

```diff
@@ -742,6 +746,11 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if args.watch && isSTS && mode == aws.ModeManual {
+		r.Reporter.Errorf("Cannot watch in mode 'manual' since the user will need to run outputted commands.")
```

Comment:
Maybe yielded instead of outputted?

Reply:
how about `Cannot watch in mode 'manual' as you will need to perform manual commands as part of the process`? but probably best to get feedback on exact phrasing from the reporter/bu